config/config.go

```diff
@@ -769,6 +790,14 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("gdpr.timeouts_ms.init_vendorlist_fetches", 0)
 	v.SetDefault("gdpr.timeouts_ms.active_vendorlist_fetch", 0)
 	v.SetDefault("gdpr.non_standard_publishers", []string{""})
+	v.SetDefault("gdpr.tcf2.enabled", true)
```

Comment:
Does it make sense to enable everything by default? Honest question. Not sure how we did it for TCF1.

Reply:
I don't think the defaults have changed since TCF1 went in. We don't have an option to turn off TCF1 completely, just what to assume if there is no GDPR signal at all. It makes sense to have everything on, as TCF1 was hardcoded with everything on.